lib/v2/learnku/topic.js

```diff
@@ -34,18 +34,25 @@ module.exports = async (ctx) => {
         link: url,
         description,
         item: list
-            .map((item) => {
+            .map(async (item) => {
                 const $ = cheerio.load(item);
                 const categoryName = $('.category-name').text().trim();
                 if (['置顶', '广告'].includes(categoryName)) {
                     return '';
                 }
                 $('.topic-title i').remove();
+                const itemLink = $('.topic-title-wrap').attr('href');
+                const itemResponse = await got({
+                    method: 'get',
+                    url: itemLink,
+                });
+                const itemData = itemResponse.data;
+                const article = cheerio.load(itemData);
                 return {
                     title: $('.topic-title').text().trim(),
+                    description: article('.article-content .content-body p').text().substring(0, 70) + '...',
```

Comment:
You can provide a fulltext description and let user use the [build-in brief parameter](https://docs.rsshub.app/parameter.html#shu-chu-jian-xun) to trim it down.

Reply:
okay